conjure-java-core/src/test/java/com/palantir/conjure/java/types/MissingFieldWireFormatTests.java

```diff
@@ -33,58 +36,66 @@
-                .hasMessage("Some required fields have not been set: [ridValue]");
+        assertThatLoggableExceptionThrownBy(() -> mapper.readValue("{}", RidExample.class))
+                .isInstanceOf(SafeIllegalArgumentException.class)
+                .hasMessage("Some required fields have not been set")
+                .hasExactlyArgs(SafeArg.of("missingFields", ImmutableList.of("ridValue")));
     }
 
     @Test
     public void missing_bearertoken_field_should_throw() throws Exception {
-        assertThatThrownBy(() -> mapper.readValue("{}", BearerTokenExample.class))
-                .isInstanceOf(IllegalArgumentException.class)
-                .hasMessage("Some required fields have not been set: [bearerTokenValue]");
+        assertThatLoggableExceptionThrownBy(() -> mapper.readValue("{}", BearerTokenExample.class))
+                .isInstanceOf(SafeIllegalArgumentException.class)
+                .hasMessage("Some required fields have not been set")
+                .hasExactlyArgs(SafeArg.of("missingFields", ImmutableList.of("bearerTokenValue")));
     }
 
     @Test
     public void missing_datetime_field_should_throw() throws Exception {
-        assertThatThrownBy(() -> mapper.readValue("{}", DateTimeExample.class))
-                .isInstanceOf(IllegalArgumentException.class)
-                .hasMessage("Some required fields have not been set: [datetime]");
+        assertThatLoggableExceptionThrownBy(() -> mapper.readValue("{}", DateTimeExample.class))
+                .isInstanceOf(SafeIllegalArgumentException.class)
+                .hasMessage("Some required fields have not been set")
+                .hasExactlyArgs(SafeArg.of("missingFields", ImmutableList.of("datetime")));
     }
 
     @Test
     public void missing_double_field_should_throw() throws Exception {
-        assertThatThrownBy(() -> mapper.readValue("{}", DoubleExample.class))
-                .isInstanceOf(IllegalArgumentException.class)
-                .hasMessage("Some required fields have not been set: [doubleValue]");
+        assertThatLoggableExceptionThrownBy(() -> mapper.readValue("{}", DoubleExample.class))
+                .isInstanceOf(SafeIllegalArgumentException.class)
+                .hasMessage("Some required fields have not been set")
+                .hasExactlyArgs(SafeArg.of("missingFields", ImmutableList.of("doubleValue")));
     }
 
     @Test
     public void missing_binary_field_should_throw() throws Exception {
-        assertThatThrownBy(() -> mapper.readValue("{}", BinaryExample.class).getBinary())
-                .isInstanceOf(IllegalArgumentException.class)
-                .hasMessage("Some required fields have not been set: [binary]");
+        assertThatLoggableExceptionThrownBy(() -> mapper.readValue("{}", BinaryExample.class).getBinary())
+                .isInstanceOf(SafeIllegalArgumentException.class)
+                .hasMessage("Some required fields have not been set")
```

Comment:
After safe-logging is update this assertion will fail because it checks the rendered throwable message rather than the safe log message.